app/lib/common/widgets/html_editor/services/mention_shortcuts.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:acter/common/widgets/html_editor/components/mention_menu.dart';
+import 'package:acter/common/widgets/html_editor/models/mention_type.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+List<CharacterShortcutEvent> mentionShortcuts(
+  BuildContext context,
+  String roomId,
+) {
+  return [
+    CharacterShortcutEvent(
+      character: userMentionChar,
+      handler: (editorState) => _handleMentionTrigger(
+        context: context,
+        editorState: editorState,
+        type: MentionType.user,
```

Comment:
like _here_ is where we'd already decided upon which of the inner listing widgets to use... no need for `MentionType` at all I think ...